MapView/Map/RMShape.h

```diff
@@ -78,7 +78,7 @@
 
 @property (nonatomic, assign) BOOL scaleLineWidth;
 @property (nonatomic, assign) CGFloat shadowBlur;
-@property (nonatomic, assign) CGSize shadowOffset;
+@property (assign) CGSize shadowOffset;
```

Comment:
It's shadowing an atomic property on `CALayer`. Surprised nobody else saw it and got irritated by it.